server/internal/graphql/resolver/meshsync.go

```diff
@@ -65,29 +68,22 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider,
 			dbHandler.Lock()
 			defer dbHandler.Unlock()
 
-			r.Log.Info("Dropping Meshery Database")
-			err = dbHandler.Migrator().DropTable(
-				&meshsyncmodel.KeyValue{},
-				&meshsyncmodel.Object{},
-				&meshsyncmodel.ResourceSpec{},
-				&meshsyncmodel.ResourceStatus{},
-				&meshsyncmodel.ResourceObjectMeta{},
-				&models.PerformanceProfile{},
-				&models.MesheryResult{},
-				&models.MesheryPattern{},
-				&models.MesheryFilter{},
-				&models.PatternResource{},
-				&models.MesheryApplication{},
-				&models.UserPreference{},
-				&models.PerformanceTestConfig{},
-				&models.SmiResultWithID{},
-				models.K8sContext{},
-			)
+			tables, err := dbHandler.Migrator().GetTables()
 			if err != nil {
 				r.Log.Error(err)
 				return "", err
 			}
 
+			r.Log.Info("Dropping Meshery Database")
+
+			for _, table := range tables {
+				err = dbHandler.Migrator().DropTable(table)
+				if err != nil {
+					r.Log.Error(err)
+					return "", errors.New(ErrDatabaseOpenCode, errors.Alert, []string{"Database is unreachable"}, []string{err.Error()}, []string{"Database connection may have been lost"}, []string{"Make sure your database is reachable"})
```

Comment:
wrap this errors.New call inside a Meshkit error function `ErrDatabaseOpen` like we have at other places. And also Pass `ErrDatabaseOpen` in r.Log.Error() above.